### ISSUE TITLE
Note Python version in showcase

### ISSUE THREAD

**Issue by smontanaro** [python/pythondotorg#290]
We received the following note at webmaster@python.org from a Python 2.7 user who has presumably not yet tried Python 3.x:
# 

In the homepage, the first slide of the showcase says

```
>>> 17 / 3
5.6666666

>>> 17 / 3  # classic division returns a float
5.666666666666667
```

but 17/3 in python returns just 5, does not return a float. Is this related
to a specific python version? I am using python 2.7. Please help me clarify
this.
# _A note in the footer of the showcase would be useful too._

I pass his (highlighted) request along, so it gets in the mill. It's not clear there's a place to stick such a reference, or that it's really necessary. Maybe you could be subtle and slip an evaluation of sys.version into the showcase. :-)

**Comment by smontanaro** [comment]
Apologies for the shouting. It seems Github's markdown implementation is badly broken.

**Comment by jhogue** [comment]
All... @frankwiles asked me to take a look at this issue and determine if the was something that should be added to the index template or the better done in the content. I ended up deciding that it would be best handled by the content in the examples. Since I had a log in for the Django portion of the site, I went ahead and updated them. Here is what I did: 

• All code samples (left side): Added "Python 3" to the first comment. 
• All code sample descriptions (right side): Since the links at the end of each description go to docs for Python 3, I updated the language in the link to make that clear. 

I hope this is acceptable. If someone wants to review my answer here and my work at http://www.python.org, this issue can be closed.
